concent_api/core/payments/bankster.py

```diff
@@ -21,14 +21,16 @@
 from common.helpers import get_current_utc_timestamp
 from common.logging import log
 from core.constants import ETHEREUM_ADDRESS_LENGTH
-from core.exceptions import BanksterTimestampError
+from core.exceptions import PaymentTimestampError
 from core.exceptions import TooSmallProviderDeposit
 from core.models import Client
 from core.models import DepositAccount
 from core.models import DepositClaim
 from core.models import Subtask
 from core.payments import service
+from core.payments.backends.sci_backend import get_youngest_payment_timestamp_from_subtask_results_accepted_list
 from core.payments.backends.sci_backend import TransactionType
+from core.payments.service import validate_that_last_closure_time_is_older_than_oldest_payment
```

Comment:
You can use this function from import which is 3 lines higher

Reply:
ok, but now it is imported from `core.payments.validation`